src/m365/planner/commands/task/task-reference-remove.ts

```diff
@@ -0,0 +1,141 @@
+import * as _ from 'lodash';
```

Comment:
While it's not needed in this particular case, because we'll be removing reference to lodash here anyway, in the future, when using a package, let's ensure that we have it referenced as a dependency in package.json and npm-shrinkwrap. Right now, it's not the case. I think it's working, because lodash is a child dependency of inquirer but if we replaced inquirer with a different, it would break CLI here.

Reply:
Totally forgot to check that. I will take this into account in the future 😄